app/Http/Resources/User.php

```diff
@@ -18,6 +18,7 @@ public function toArray($request)
             'name'          => $this->name_private,
             'name_private'  => $this->name_private,
             'avatar'        => $this->resolveAvatarUrl(),
+            'discord_id'    => $this->discord_id,
```

Comment:
Does this need to go out with an API response? I don't think it would be needed for anything and would expose an internal ID

Reply:
I'm not sure either. I work on a discord bot at the same time and having the ID of my users can make interactions for this bot simpler (I know Taylor is also working on something similar). I don't really know how sensitive the `discord_id` is because you can copy anyone's ID directly from the discord application, so I don't feel like I'm divulging anything very important.

Reply:
I agree with @arthurpar06 here, Discord ID's are not private. You can just right click on a user on any server and get their ID's. And it is not used somewhere critical as far as I know.

Imagine it like our user id field, nothing special and not useful at all externally. But it can be useful to have it returned with api calls 'cause people can develop stuff internally (maybe with their discord bots, landing sites etc)